internal/tiltfile/docker.go

```diff
@@ -325,34 +325,40 @@ func (s *tiltfileState) reposToDomain(image *dockerImage) []model.LocalGitRepo {
 		})
 	}
 
+	return result
+}
+
+func (s *tiltfileState) reposToDomain(image *dockerImage) []model.LocalGitRepo {
+	var paths []localPath
 	for _, m := range image.mounts {
-		maybeAddRepo(m.src)
+		paths = append(paths, m.src)
 	}
-	maybeAddRepo(image.baseDockerfilePath)
-	maybeAddRepo(image.staticDockerfilePath)
-	maybeAddRepo(image.staticBuildPath)
-	maybeAddRepo(image.tiltfilePath)
+	paths = append(paths,
+		image.baseDockerfilePath,
+		image.staticDockerfilePath,
+		image.staticBuildPath,
+		image.tiltfilePath)
 
-	return result
+	return pathsToRepos(paths)
 }
 
-func (s *tiltfileState) dockerignoresToDomain(image *dockerImage) []model.Dockerignore {
+func pathsToDockerIgnores(paths []string) []model.Dockerignore {
```

Comment:
same comment as `pathsToRepos`

also, nit, your capitalization of `dockerIgnore` is inconsistent.